src/App.vue

```diff
@@ -48,9 +47,9 @@
                                           v-on:css-errors="cssErrors = $event"/>
                   </div>
               </div>
-            </div>
+            </b-col>
 
-            <div class="data-container col-4 border overflow-auto pr-0 pl-0">
+            <b-col cols="4" class="data-container border overflow-auto pr-0 pl-0 rounded">
```

Comment:
This should only need the card wrapper! Why are we adding all of these classes???

Reply:
https://getbootstrap.com/docs/4.3/components/card/